CHANGELOG.md

```diff
@@ -10,7 +10,32 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 - [#288] (https://github.com/HewlettPackard/terraform-provider-oneview/issues/288) Terraform crashes on Logical Enclosure create
 - [#295] (https://github.com/HewlettPackard/terraform-provider-oneview/issues/295) server profile update in-place.
 
-
+### Oneview Features supported
+- Ethernet Network
```

Comment:
Hop Deployment Plan is validated. If yes, then please add it over here.

Reply:
Added